apps/condo/pages/404.tsx

```diff
@@ -64,4 +65,7 @@ const Error404Layout = (props): React.ReactElement => <PosterLayout
 />
 
 Custom404.container = Error404Layout
-Custom404.isError = true
\ No newline at end of file
+Custom404.isError = true
+Custom404.skipUserPrefetch = true
```

Comment:
🤔 What's this?

Reply:
you should specify a flag to not request user data. For example, on the error page they are not needed

Reply:
Hm, ok. But i personally like solution with props (like [SKIP_RESIDENT_PROP_NAME]: true) more

Reply:
Since page.something is kinda legacy (we'll be overwritten anyway in server-components (modern Next) world)